README.md

```diff
@@ -48,20 +48,19 @@ As a result of this command you get access to `mongoose_push:release` docker ima
 docker run -it --rm mongoose_push:release foreground
 ```
 
-Docker image that you have just built, exposes the port `8443` for the REST API of
-MongoosePush. Also there is a `VOLUME` for path */opt/app* where the whole MongoosePush release is stored. This volume will be handy for injecting `APNS` and REST API certificates since by default the
-docker image comes with fake, self-signed certificates.
+The docker image that you have just built, exposes the port `8443` for the HTTP API of
```

Comment:
The docker image that you have just built, exposes the port `8443` for the HTTP API of MongoosePush. 
Also there is a `VOLUME` for path */opt/app* - it is handy for injecting `APNS` and `HTTP API` certificates since by default the docker image comes with test, self-signed certificates.